certified-connectors/OrbusInfinity/apiProperties.json

```diff
@@ -0,0 +1,70 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oAuthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "scopes": [
+            "user_impersonation"
+          ],
+          "redirectMode": "GlobalPerConnector",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "{resourceUri}",
+            "IsOnbehalfofLoginSupported": true
+          },
+          "customParameters": {
+            "LoginUri": {
+              "value": "https://login.microsoftonline.com"
+            },
+            "TenantId": {
+              "value": "common"
+            },
+            "EnableOnbehalfOfLogin": {
+              "value": false
+            }
+          },
+          "clientId": "<<Please add your clientID here>>"
+        },
+        "uiDefinition": null
+      },
+      "token:tenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
```

Comment:
this should be "hidden": true

Reply:
When I try to put true without double quote got problem 
![2024-01-11_14h58_43](https://github.com/microsoft/PowerPlatformConnectors/assets/149225690/07a02538-910f-4566-a165-b66e3f2f122c)